chainerx_cc/chainerx/cuda/cuda_device.h

```diff
@@ -109,6 +109,7 @@ class CudaDevice : public Device {
     // activation.cu
 
     void IfLessElseASSA(const Array& x1, Scalar x2, Scalar pos, const Array& neg, const Array& out) override;
+    void IfLessElseAAAA(const Array& x1, const Array& x2, const Array& pos, const Array& neg, const Array& out) override;
```

Comment:
This member function seems unused. Could you remove this?

Reply:
Sure! Working on it!